examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2988,7 +2988,13 @@ class GLTFParser {
 
 			texture.flipY = false;
 
-			texture.name = textureDef.name || sourceDef.name || sourceDef.uri || '';
+			texture.name = textureDef.name || sourceDef.name || '';
+
+			if ( texture.name === '' && sourceDef.uri.length > 0 && sourceDef.uri.startsWith( 'data:image/' ) === false ) {
```

Comment:
What is this `sourceDef.uri.length > 0` check for? And [`sourceDef.uri` is optional (if image data is defined in `sourceDef.bufferView`)](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-image) so accessing `sourceDef.uri.length` can cause an exception in that case.